docs/sources/tempo/troubleshooting/alloy.md

```diff
@@ -33,6 +34,24 @@ exporter_sent_spans_ratio_total
 exporter_send_failed_spans_ratio_total
 ```
 
+If you are using Grafana Alloy to send traces to Grafana Cloud, the metrics are visible at
+`http://localhost:12345/metrics`.
```

Comment:
```suggestion
Alloy has a Prometheus scrape endpoint, `/metrics`, that you can use to check metrics locally by opening a browser to `http://localhost:12345/metrics`.
```

Reply:
Yep, that looks good :+1: I think that's clear and avoids all confusion with Grafana Cloud